assets/scripts/dialogs/SignInDialog.jsx

```diff
@@ -70,6 +73,25 @@ export default class SignInDialog extends React.Component {
     goTwitterSignIn()
   }
 
+  handleMagicLinkClick = (event) => {
+    event.preventDefault()
+
+    // Since this is not the native form submission, we manually trigger the native validation.
+    // Note: IE11 doesn't support `reportValidity()`. If IE11 users enter a bad email address and
+    // click the magic link button, they'll get an endless spinner with no error.
```

Comment:
@louh I wanted to draw attention to this.

Reply:
Never mind, I just found [this](https://streetmix.readthedocs.io/en/latest/technical/project/#browser-support). 🎉 I'll remove this note.